cypress/integration/spec_balances_amounts.js

```diff
@@ -63,33 +63,171 @@ describe('Test the rendering of balances and amounts', () => {
+        cy.get('#wallet-amount_available').should('have.text', '20.00000000 tBTC') 
+        cy.get('#wallet-amount_available').find('.thousand-digits-in-btc-amount').children().each((element) => {
+            cy.wrap(element).should('have.text', '0') 
+            cy.wrap(element).should('not.be.visible') 
+            cy.wrap(element).should('be.hidden') 
+        });
+        cy.get('#wallet-amount_available').find('.last-digits-in-btc-amount').children().each((element) => {
+            cy.wrap(element).should('have.text', '0') 
+            cy.wrap(element).should('not.be.visible') 
+            cy.wrap(element).should('be.hidden') 
+        });
+
         cy.get('#recipient_0').find('#address').invoke('val', 'bcrt1qvtdx75y4554ngrq6aff3xdqnvjhmct5wck95qs') 
         cy.get('#recipient_0').find('#amount').type(0.05, { force: true })
         cy.get('#toggle_advanced').click()
         cy.get('.fee_container').find('#fee_option_manual').click()
         cy.get('#fee_manual').find('#fee_rate').clear( { force: true })
         cy.get('#fee_manual').find('#fee_rate').type(5, { force: true }) // Should be a fee of 709 sats.
         cy.get('#create_psbt_btn').click()
+        // check that the psbt amount is displayed correct
+        cy.get('#psbt-amount-0').should('have.text', '0.05000000 tBTC')  
+        cy.get('#psbt-amount-0').find('.thousand-digits-in-btc-amount').children().each((element) => {
+            cy.wrap(element).should('have.text', '0') 
+            cy.wrap(element).should('be.hidden') 
+            cy.wrap(element).should('not.be.visible') 
+        });
+        cy.get('#psbt-amount-0').find('.last-digits-in-btc-amount').children().each((element) => {
+            cy.wrap(element).should('have.text', '0') 
+            cy.wrap(element).should('be.hidden') 
+            cy.wrap(element).should('not.be.visible') 
+        });
+
         cy.get('body').contains("Paste signed transaction")
         cy.get('#satoshis_hot_keys_tx_sign_btn').click()
         cy.get('#satoshis_hot_keys_hot_sign_btn').click()
         cy.get('#hot_enter_passphrase__submit').click()
         cy.get('#broadcast_local_btn').click()
+    })
+
+
+    it('Sending a tx with 0.001234 tBTC', () => {
+        // get the ghot receive address to the clipboard
+        // cy.selectWallet('Ghost wallet')
+        // cy.selectWallet('Ghost wallet')       // Once again because only once doesn't work for some stupid unknown reason
+        // cy.get('#btn_receive').click()
+        // cy.get('#address').click()  // get receive address from ghost wallet 
+        cy.selectWallet('Funding wallet')
+        cy.selectWallet('Funding wallet')       // Once again because only once doesn't work for some stupid unknown reason
+        cy.get('#btn_send').click()
+        // check that the amount_available for sending is displayed correct
+        // cy.get('#wallet-amount_available').should('have.text', '20.00000000 tBTC') 
```

Comment:
are these tests relevant? kill commented code or make em pass?

Reply:
Done in  42ae0a2e